app/portal/courses/[id]/page.js

```diff
@@ -98,14 +75,111 @@ export default function Page() {
   // Selected Course Data
   const [selectedCourse, setSelectedCourse] = useState(null)
   const [selectedMaterials, setSelectedMaterials] = useState(null)
+  const [formState, setFormState] = useState(initialState)
   const handleOpenMaterialNew = () => setOpenMaterialNew(true)
   const handleCloseMaterialNew = () => setOpenMaterialNew(false)
   const { courses, course_materials } = useData();
+  const axiosInstance = AxiosWithAuth()
 
   const pathname = usePathname()
   const regex = /-/g
   const newStr = pathname.slice(16).replace(regex, " ")
 
+  const getMaterialType = (material) => {
+    switch (material.material_type_id) {
+      case 1:
+        material.material_type = "Document"
+        break;
+      case 2:
+        material.material_type = "Presentation"
+        break;
+      case 3:
+        material.material_type = "Quiz"
+        break;
+      case 4:
+        material.material_type = "Video"
+        break;
+      default:
+        console.log('No Material Type detected')
+    }
+    return material
+  }
+
+  const sendNewMaterialData = (materialData) => {
+    axiosInstance
+      .post(`${process.env.NEXT_PUBLIC_BE_API_URL}/courseMaterials/create`, materialData)
+      .then((res) => {
+        console.log(res)
+        setSelectedMaterials([
+          ...selectedMaterials,
+          getMaterialType(res.data[0])
+        ])
+        setOpenMaterialNew(false)
+        setFormState(initialState)
+      })
+      .catch((err) => {
+        console.log(err)
+      })
+  }
+
+  // "Document" - 1,
```

Comment:
Did you still need these comments @paulstgermain ?

Reply:
I definitely do not, looks like I forgot to commit my change in removing them! Thanks for catching this!